.github/workflows/pull-translations.yml

```diff
@@ -0,0 +1,74 @@
+name: Pull & Commit Translations from Mojito
+
+on:
+  schedule:
+    # Run every day at 9am and 9pm ET.
+    # Note: We only run every 12 hours, otherwise our Mojito webapp (on Heroku)
+    # wouldn't be able to sleep, and would go over its free hours.
+    - cron: '0 13,1 * * *'
+
+jobs:
+  pull-translations:
+    if: github.repository_owner == 'patcon'
```

Comment:
To keep this workflow from running on the upstream repo once a PR was created. We should remove only when we're ready to merge.

```suggestion
```